### PR TITLE
Several code improvements

### PR DESCRIPTION
**Pull request by frankiek3**
Several code improvements that should help speed up host of mod.

### CODE REVIEW

File: Modules/Utils.cs [hunk 1]
```diff
@@ -535,29 +541,32 @@ public static void NotifyRoles(bool isMeeting = false, PlayerControl SpecifySeer
...
                 }
+
                 //Markとは違い、改行してから追記されます。
                 string SelfSuffix = "";
 
-                if (seer.Is(CustomRoles.BountyHunter) && seer.GetBountyTarget() != null)
+                //他人用の変数定義
+                bool SeerKnowsImpostors = false; //trueの時、インポスターの名前が赤色に見える
+
+                if (seer.Is(CustomRoles.BountyHunter))
```
Comment (soukunsandesu) [inline Modules/Utils.cs:551]: Please change to switch.

example)

switch (seer.GetCustomRole())
{
　case CustomRoles.BountyHunter:
　　　　break;
}

File: Modules/Utils.cs [hunk 2]
```diff
@@ -172,6 +177,7 @@ public static bool HasTasks(GameData.PlayerInfo p, bool ForRecompute = true)
                     if (cSubRole == CustomRoles.Lovers)
                     {
                         //ラバーズがクルー陣営の場合タスクを付与しない
+                        // A Crew task win does not require Lovers tasks
```
Comment (soukunsandesu) [inline Modules/Utils.cs:180]: Not necessary as it is already written in Japanese.

**Comment by soukunsandesu** [comment]
Please change target branch “develop”.

File: Modules/Utils.cs (hunk 2 above)
Comment (frankiek3) [reply Modules/Utils.cs:180]: Ok

File: Modules/Utils.cs (hunk 1 above)
Comment (frankiek3) [reply Modules/Utils.cs:551]: Will do

**Pull request closed without merging**